### ISSUE TITLE
Not recognizing --config path argument on Windows

### ISSUE THREAD

**Issue by Robotex** [inspircd/inspircd#1736]
**Description**

On windows launching inspircd.exe with --config parameter doesn't recognize the path parameter

**Steps to reproduce the issue:**

1. Build on RelWithDebug target
2. Navigate to inspircd directory in command prompt
3. Run `inspircd.exe --config inspircd.conf` (or any other path, either relative or absolute)

**Describe the results you received:**
```
Unknown parameter 'inspircd.conf'
Usage: inspircd.exe [--nofork] [--nolog] [--nopid] [--debug] [--config <config>]
                    [--runasroot] [--version]
```

**Describe the results you expected:**
IRC daemon launching with the configurations read from given path

**Additional information you deem important (e.g. issue happens only occasionally):**

**Output of `./bin/inspircd --version`:**
`InspIRCd-3.4.0`

**Comment by SaberUK** [comment]
This is a duplicate of #546. 

Unfortunately the getopt_long implementation we use doesn't support it for some reason.

**Comment by Robotex** [comment]
That issue is dated 2013, I thought it was fixed in 2.2? If it wasn't fixed, shouldn't it be removed from accepted parameters on Windows or atleast write a note in the documentation? I spent an hour trying to figure out why it didn't take the path parameter.

Also if the getopt_long implementation is giving issues, why not use a third party drop in replacement? There are plenty of open source implementations, for example [ya_getopt](https://github.com/kubo/ya_getopt)
